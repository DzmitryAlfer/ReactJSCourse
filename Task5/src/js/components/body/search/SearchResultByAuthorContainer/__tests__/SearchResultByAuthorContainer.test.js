import React from 'react';
import Renderer from 'react-test-renderer';
import {SearchResultByAuthorContainer} from '../index'

jest.mock('../../../../../services/api/MoviesDataSource');

describe('SearchResultByAuthorContainer', () => {
    test('Component render', () => {

        const tree = Renderer.create(
            <SearchResultByAuthorContainer author='Show all'/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});