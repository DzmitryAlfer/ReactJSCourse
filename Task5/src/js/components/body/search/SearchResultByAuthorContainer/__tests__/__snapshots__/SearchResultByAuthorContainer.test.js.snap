// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultByAuthorContainer Component render 1`] = `
<div
  className="search-results"
/>
`;
